### PR TITLE
Prepare release 0.10.0-beta.1

### LINKED ISSUE #645: Draft release
* GitHub release
* Explain how the gateway works and the gateway features in short
* Say in one sentence that it requires the anchor
* Update changelog with link to release
* Link from release to changelog
* Copy changelog to release

### PR DESCRIPTION
**Pull request by schemar**
* Added a release description
* Reordered the change log
* Updated keywords of `package.json`
* Updated version of `package.json`
* License is now valid SPDX expression

Fixes #645

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,55 +2,70 @@
...
-* Ported repository to solidity 0.5.0 ([#480](https://github.com/OpenSTFoundation/mosaic-contracts/pull/480)).
-* Objects are now hashed according to [EIP 712](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md) ([#399](https://github.com/OpenSTFoundation/mosaic-contracts/pull/399), [#566](https://github.com/OpenSTFoundation/mosaic-contracts/pull/566)).
+  * Core logic is now cleanly split into a "mosaic core" and an "anchor" ([#522](https://github.com/OpenSTFoundation/mosaic-contracts/pull/522), [#549](https://github.com/OpenSTFoundation/mosaic-contracts/pull/549)).
 * Gateway can now interact with decentralized mosaic core ([#463](https://github.com/OpenSTFoundation/mosaic-contracts/pull/463)).
-* Contracts can now be `Organized` in line with [openst-contracts](https://github.com/OpenSTFoundation/openst-contracts) ([#513](https://github.com/OpenSTFoundation/mosaic-contracts/pull/513)).
-  * Gateways are now `Organized` ([#515](https://github.com/OpenSTFoundation/mosaic-contracts/pull/515)).
+* Gateways are now message-based ([#293](https://github.com/OpenSTFoundation/mosaic-contracts/pull/293)).
```
Comment (benjaminbollen) [inline CHANGELOG.md]: half a years work, summarised in a single bulletpoint cc @deepesh-kn 🥳

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,55 +2,70 @@
 
 ## Version 0.10.0 ⚓️ Anchor
 
-[**Release v0.10.0, (date goes here)**](Link to release goes here)
+<!-- [**Release 0.10.0, (<date-here>)**](https://github.com/OpenSTFoundation/mosaic-contracts/releases/tag/0.10.0) -->
+
+A gateway for a given EIP20 token is comprised of a `EIP20Gateway` contract on origin  and a corresponding `EIP20CoGateway` contract on auxiliary.
+On auxiliary there is an ERC20 utility token contract that mints and burns utility tokens.
```
Comment (deepesh-kn) [inline CHANGELOG.md]: EIP20 tokens instead of ERC20?

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,55 +2,70 @@
...
+It atomically mirrors tokens staked and unstaked on the origin chain.
+
+Atomicity is achieved using a 2-phase message passing architecture between the chains.
+Messages are declared on the source chain and confirmed on the target chain.
+In order to confirm a message, a facilitator uses a Patricia Merkle proof once the source chain's state has been transferred to the target.
```
Comment (deepesh-kn) [inline CHANGELOG.md]: Merkle Patricia proof instead of Patricia Merkle proof?

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,55 +2,70 @@
...
+Once messages are confirmed on the target chain, the facilitator can efficiently progress them by providing the hash lock secret.
```
Comment (deepesh-kn) [inline CHANGELOG.md]: Should we also mention about the progress with proof?

File: CHANGELOG.md (hunk 2 above)
Comment (schemar) [reply CHANGELOG.md]: Will update

File: CHANGELOG.md (hunk 3 above)
Comment (schemar) [reply CHANGELOG.md]: True!

File: CHANGELOG.md (hunk 4 above)
Comment (schemar) [reply CHANGELOG.md]: Will add

**Pull request merged**